spring-boot-servlet/src/main/java/com/baeldung/props/Constants.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.props;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Properties;
+
+public final class Constants {
```

Comment:
Is this class still being used? I didn't see any references to it or any of the public properties.